src/FrameManager.ts

```diff
@@ -122,6 +119,10 @@ class FrameManager {
   }
 
   private _expectedClientOrigin(): string {
+    if (this._frameLocation === 'about:blank') {
+      return '';
+    }
```

Comment:
I think I understand the intent here: don't send messages to `about:blank`.  I considered that restriction, but decided that it was probably unnecessary since `about:blank` should just be an empty page.  I would suggest returning `null` instead of the empty string though. That would make it clearer in the type signatures that `FrameManager` has an expected state which is "No valid origin"

Reply:
There was some case where I ran into this... I think it was on the demo app?  It was throwing an error which i figured we didn't want to do.  I'll change it to null and make sure thats anticpated.

Reply:
Oh, nice catch. I didn't expect it to throw errors when on about:blank. Shows what I know.